modules/visual_script/visual_script_nodes.cpp

```diff
@@ -1080,10 +1112,13 @@ class VisualScriptNodeInstanceOperator : public VisualScriptNodeInstance {
 			if (p_outputs[0]->get_type() == Variant::STRING) {
 				r_error_str = *p_outputs[0];
 			} else {
+				// TODO: Do we need the long names here? we should use Variant::get_operator_name()
+				//       instead of VisualScriptOperator::get_operator_name(). That will also make
+				//       it consistent with gdscript error messages.
```

Comment:
I don't have a strong opinion, but yes if you think it's best to use the `Variant` names in the error that should be fine.

Reply:
Thanks. Yeah, I think Variant is better fit here, I will update the PR with this change today.